lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1058,25 +1196,12 @@ void PatchEntryPointMutate::addUserDataArgs(SmallVectorImpl<UserDataArg> &userDa
         userDataValue = DescRelocMagic | node.innerTable[0].set;
       }
       // Add the arg (descriptor set pointer) that we can potentially unspill.
-      userDataSize = addUserDataArg(userDataArgs, userDataValue, node.sizeInDwords, &descSetUsage.entryArgIdx,
-                                    useFixedLayout, userDataSize, builder);
+      auto &descSetUsage = userDataUsage->descriptorSets[node.innerTable[0].set];
+      addUserDataArg(userDataArgs, userDataValue, node.sizeInDwords, &descSetUsage.entryArgIdx, builder);
```

Comment:
This still crashes as `userDataUsage->descriptorSets` is empty.
IIRC you said it should pass a `nullptr` to `addUserDataArg` if the descriptor set is not known.